src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit } from '@angular/core';
+
+import { PoolService } from '../../../shared/api/pool.service';
+import { CdTableColumn } from '../../../shared/models/cd-table-column';
+import { CdTableSelection } from '../../../shared/models/cd-table-selection';
+
+@Component({
+  selector: 'cd-pool-list',
+  templateUrl: './pool-list.component.html',
+  styleUrls: ['./pool-list.component.scss']
+})
+export class PoolListComponent implements OnInit {
+  pools = [];
+  columns: CdTableColumn[];
+  selection = new CdTableSelection();
+
+  constructor(
+    private poolService: PoolService,
+  ) {
+    this.columns = [
+      {prop: 'pool_name', name: 'Name', flexGrow: 2},
+      {prop: 'application_metadata_array', name: 'Applications'},
```

Comment:
I've created a pool, and enabled the rbd application, but I can't see this information on the UI (applications columns is empty, and not application info is displayed on `Details` tab).

`ceph osd pool application enable rbd rbd`

Reply:
Have you reloaded the dashboard? There are backend changes